pkg/frontend/openshiftcluster_putorpatch.go

```diff
@@ -285,5 +286,21 @@ func (f *frontend) ValidateNewCluster(ctx context.Context, subscription *api.Sub
 	if err != nil {
 		return err
 	}
-	return f.quotaValidator.ValidateQuota(ctx, f.env.Environment(), f.env, subscription.ID, subscription.Subscription.Properties.TenantID, cluster)
+
+	fpAuthorizer, err := f.env.FPAuthorizer(subscription.Subscription.Properties.TenantID, f.env.Environment().ResourceManagerEndpoint)
```

Comment:
Why is the change from spAuthorizer to fpAuthorizer needed in frontend? I have an idea, but wanted to clarify.

Reply:
I looked at the master branch and it looks like it uses fpAuthorizer still. To make it similar to QuotaValidation and since theres no fpAuthorizer for it on frontend, I added it in. I moved it to the function itself now, so this currently is a bit outdated.